src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -64,12 +64,6 @@
     <Reference Include="Mono.Cecil.Rocks">
       <HintPath>..\packages\Mono.Cecil.0.9.5.4\lib\net40\Mono.Cecil.Rocks.dll</HintPath>
     </Reference>
-    <Reference Include="NServiceBus.NewCompilerBits">
-      <HintPath>TestDlls\NServiceBus.NewCompilerBits.dll</HintPath>
-    </Reference>
-    <Reference Include="NServiceBus.OldCompilerBits">
-      <HintPath>TestDlls\NServiceBus.OldCompilerBits.dll</HintPath>
-    </Reference>
```

Comment:
@SimonCropp these were only used for the old compiler to check whether we correctly scan for the task name. Since I don't have the sources and it has changed from `Action` to `Func` I cannot fix this. Furthermore the code is now correctly doing it. When we compile & execute the same tests on an older compiler they would green now.